<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ContentsManagement</class>
 <widget class="QWidget" name="ContentsManagement">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>콘텐츠 관리</string>
  </property>
  <widget class="QPushButton" name="PhotoLoadDialog">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>150</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>사진 불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="VideoLoadDIalog">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>150</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>동영상 불러오기</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="MasterList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <column>
    <property name="text">
     <string>파일 이름</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>생성 시간</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>파일 타입</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>이벤트</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>사람</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>장소</string>
    </property>
   </column>
  </widget>
  <widget class="QListWidget" name="UnregisterFile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>400</width>
     <height>200</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="EventNameInput">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>340</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="PersonNameInput">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>390</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="EventName">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>320</y>
     <width>60</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>이벤트 이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="PersonName">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>370</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>인물 이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="PlaceName">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>420</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>장소 이름</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PlaceNameInput">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>440</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="Notice2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>등록 파일</string>
   </property>
  </widget>
  <widget class="QLabel" name="Notice1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>미등록 파일</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddItem">
   <property name="geometry">
    <rect>
     <x>525</x>
     <y>475</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
  </widget>
  <widget class="QLabel" name="AdditionalInformation">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>추가 정보</string>
   </property>
  </widget>
  <widget class="QLabel" name="DefaultInformation">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>기본 정보</string>
   </property>
  </widget>
  <widget class="QLabel" name="FileName">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>320</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>파일 이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="FileType">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>380</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>파일 타입</string>
   </property>
  </widget>
  <widget class="QLabel" name="Notice4">
   <property name="geometry">
    <rect>
     <x>475</x>
     <y>550</y>
     <width>250</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>추가 정보가 빈 칸일 경우 None으로 등록됩니다.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Reload">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>40</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>새로고침</string>
   </property>
  </widget>
  <widget class="QLabel" name="Notice3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>90</y>
     <width>270</width>
     <height>40</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>사진 또는 동영상을 저장 했을 경우에
새로고침 버튼을 눌러서 다시 로드를 하셔야합니다.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadFromFile">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>560</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>등록 정보 불러오기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveToFile">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>560</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>등록 정보 저장하기</string>
   </property>
  </widget>
  <widget class="QLabel" name="FileNamePrint">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>350</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="FileTypePrint">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>410</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="DeleteItem">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>520</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>선택 삭제</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MakeEmpty">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>520</y>
     <width>130</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>나눔바른고딕</family>
    </font>
   </property>
   <property name="text">
    <string>전부 삭제</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>PhotoLoadDialog</sender>
   <signal>clicked()</signal>
   <receiver>ContentsManagement</receiver>
   <slot>OpenPhotoLoadDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>549</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>681</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>VideoLoadDIalog</sender>
   <signal>clicked()</signal>
   <receiver>ContentsManagement</receiver>
   <slot>OpenVideoLoadDialog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>637</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>637</x>
     <y>191</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AddItem</sender>
   <signal>clicked()</signal>
   <receiver>ContentsManagement</receiver>
   <slot>AddItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>592</x>
     <y>508</y>
    </hint>
    <hint type="destinationlabel">
     <x>675</x>
     <y>503</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Reload</sender>
   <signal>clicked()</signal>
   <receiver>ContentsManagement</receiver>
   <slot>Reload()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>512</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MakeEmpty</sender>
   <signal>clicked()</signal>
   <receiver>ContentsManagement</receiver>
   <slot>MakeEmpty()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>520</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DeleteItem</sender>
   <signal>clicked()</signal>
   <receiver>ContentsManagement</receiver>
   <slot>DeleteItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>168</x>
     <y>532</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>509</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LoadFromFile</sender>
   <signal>clicked()</signal>
   <receiver>ContentsManagement</receiver>
   <slot>LoadFromFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>576</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>572</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>SaveToFile</sender>
   <signal>clicked()</signal>
   <receiver>ContentsManagement</receiver>
   <slot>SaveToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>367</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MasterList</sender>
   <signal>itemClicked(QTableWidgetItem*)</signal>
   <receiver>ContentsManagement</receiver>
   <slot>SelectRegisterFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>345</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>UnregisterFile</sender>
   <signal>itemClicked(QListWidgetItem*)</signal>
   <receiver>ContentsManagement</receiver>
   <slot>SelectUnregisterFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>320</x>
     <y>17</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OpenPhotoLoadDialog()</slot>
  <slot>OpenVideoLoadDialog()</slot>
  <slot>AddItem()</slot>
  <slot>Reload()</slot>
  <slot>MakeEmpty()</slot>
  <slot>DeleteItem()</slot>
  <slot>LoadFromFile()</slot>
  <slot>SaveToFile()</slot>
  <slot>SelectRegisterFile()</slot>
  <slot>SelectUnregisterFile()</slot>
 </slots>
</ui>
